### ISSUE TITLE
Users struggle to find individual contributions via the main navigation menu

### ISSUE THREAD

**Issue by nickykrause** [18F/openFEC-web-app#2061]
@nickykrause commented on [Tue May 16 2017](https://github.com/18F/fec-testing/issues/62)

While testing [this script](https://github.com/18F/fec-testing/blob/master/Testing-Scripts/Test-Script_2017-05-12.md), one user struggled to find how she could search `Individual contributions.` 

Specifically, she was answering this question: _If you wanted to know who one of your neighbors or family members (or any specific person) donated money to in the last presidential election, what would you do to find out?_

Her first instinct was to go to the Campaign finance data menu, but she didn't see something there that she liked. While she was clicking around, she said _"I am just looking for the word `individual`..."_

Eventually, she scrolled down on the homepage and saw the word "individual" in the Campaign finance data section.  

The menu:
<img width="1258" alt="screen shot 2017-05-16 at 10 31 18 am" src="https://cloud.githubusercontent.com/assets/1450173/26114423/d65e16d8-3a22-11e7-8ba9-dc32903f48b5.png">

---

@nickykrause commented on [Mon May 22 2017](https://github.com/18F/fec-testing/issues/62#issuecomment-303121451)

We tested this again last week. Two of the three participants again struggled to find Individual Contributions, which suggests that this is indeed a problem. For both of the two users who struggled, their inclination was to go to the CFD menu, but they didn't see anything there that seemed relevant to them. 

In both cases, they said they were looking for something like "Individual" or "Donor history"

---

@noahmanger commented on [Mon May 22 2017](https://github.com/18F/fec-testing/issues/62#issuecomment-303128137)

This is something I'd like to move on sooner than later. Can you start doing some thinking around solutions?

---

@nickykrause commented on [Mon May 22 2017](https://github.com/18F/fec-testing/issues/62#issuecomment-303132030)

Sure @noahmanger . Should we move this one into one of the other repos? I'm happy to move it and assign myself (which repo makes the most sense?). I can take a look at it this week. 

---

@noahmanger commented on [Mon May 22 2017](https://github.com/18F/fec-testing/issues/62#issuecomment-303136493)

Go for it!

**Comment by nickykrause** [comment]
Connecting this issue back to the original issue where the new menu design was refined: https://github.com/18F/fec-style/issues/655

**Comment by nickykrause** [comment]
I have thought a lot about this today and have many versions of possible designs for updates to the navigation. I have scheduled some time tomorrow to chat with @jenniferthibault about this, because I think it will be easier to discuss, but I am going to summarize some of my thoughts here and show just one set of the designs that are currently my favorite.

### The main thoughts/observations influencing the proposed design
🤔 1. Currently, we collect almost everything in the data menu under the heading of `advanced data`, and I am reflecting on whether or not users will know what that means, and whether or not it is the most useful way to organize the information in that wayfinding/scanning moment.
> 💡 Organize the information differently (vague, I know)

---
 
🤔 2. People are struggling to find individual contributions because it is hidden within `receipts`
> 💡 Make individual contributions easier to find

---

🤔 3. Some users have expressed hesitation about the site-wide search on the grounds that they don't "trust" site-wide searches, but they have not expressed (at least in the sessions I've run) concern with the CFD search or the legal search. Users seem to view these searches differently (possibly as databases) than they view the site-wide search, and they often seem to prefer them.
> 💡 Make the "database searches" easier to reach

---

🤔 4. Users who go to the data menu and select `Candidates` or `Committees`ultimately end up wanting to just search for a specific candidate or committee. They end up on the advanced data table and do their search there, but then don't always end up on the candidate/committee profiles that would be of use to them.
> 💡 Make the candidate and committee search easier to reach

---

🤔 5. Some users have not known that the section "home" links are indeed links. It has been unclear from testing whether or not this is actually problematic, but several users have commented that those links "don't look clickable." 
> 💡 Make the home links look more like links

---

🤔 6. A small group of users have not seemed to notice that the `Help for candidates and committees` menu contains more than just trainings, dates and deadlines, and forms. See https://github.com/18F/fec-testing/issues/67. However, it is not clear if this is actually a problem, since these users often end up finding the information they need in other ways.
> 💡 Consider adding links to the different group types in the list of menu options.

---

### Proposed design
![navigation](https://cloud.githubusercontent.com/assets/1450173/26376792/61c9e534-3fd3-11e7-9931-71ac67f21e43.png)

---

### Feedback & next steps

I know a lot of thought went into the previous round of the nav design, and I don't want to overlook that. So, as I said earlier, I set up time to chat about this with @jenniferthibault and possible @noahmanger, and I've linked the relevant design-history-issue to this GitHub issue. 

We have 5 users signed up for testing this week Thursday and Friday, so I think it'd be interesting to test some options with them, if we can.

**Comment by nickykrause** [comment]
One more update: This menu revision does not yet address the possible issues with the location of the `Elections` link. 

As [this issue](https://github.com/18F/fec-testing/issues/57) explains, we have mounting evidence to suggest that people to not expect `elections` information to live in the CFD menu.

However, we are also planning to conduct some work soon to begin updating design of `Elections` data, and it is possible that with a more robust design, the information will deserve its own home or seem to more logically fit somewhere else. Since that is still open-ended, I have left the elections info within CFD.

**Comment by nickykrause** [comment]
Okay, after a long and very helpful chat with @noahmanger and @jenniferthibault, I am recalibrating on this issue a bit and reigning it in. 

Before I do that, though, it is worth summarizing how this `individual contributions` issue and some other, related issues have raised bigger-picture questions that probably constitute an epic-sized design task that we should address the future. That summary is below, but feel free to skip straight to the "This specific issue" section for next steps on the `individual contributors` question.

---

### Summary of bigger picture question (which we are tabling for now)
The thinking I did and the designs that I proposed are biting off a lot. Along with this issue about the findability of `individual contributions`, we have several other navigation/structural issues circulating: 

- (a) the findability of `elections` info (https://github.com/18F/fec-testing/issues/57)

- (b) the findability of compliance guidance for specific groups (https://github.com/18F/fec-testing/issues/67) 

- (c) users' apparent conflation of the H4CC section and legal section when answering "Can I do X?" questions (https://github.com/18F/fec-testing/issues/19)

When looked at together, these issues are raising some questions/concerns about the overall model that we use to organize content on the site and to help users move through it. The model we had in the past now needs to accommodate lots of new content and features that we've rapidly added leading up to the flip. So, the model may be due for some 🤔 . 
- For example, the CFD section offers a sort of "mini homepage" as a landing page, in addition to a pseudo-"index" page in the form of the `Advanced data` landing page. Meanwhile, the landing page for H4CC is sort of a hybrid of an "index" and a showcase of useful tools, and legal also has its own structure. 

It would probably make sense to take a look at how our various wayfinding components interact (homepage, main navigation, landing pages/sub-menu pages) to create a more coherent model, now that all these pieces are in place. We couldn't have done this back when H4CC didn't have much content, or before we had our homepage, or before the legal section was more established. But, perhaps now it is time.

---

### This specific issue
All of that said, for _this_ particular issue about individual contributors, I am going to reign in the task and just take a look at how individual contributions can be made more prominent via the existing CFD menu and maybe the CFD landing page, but without touching on the overall model.

**Comment by noahmanger** [comment]
Sounds good. Let's keep the bigger things in mind for the next PI planning.

**Comment by nickykrause** [comment]
Okay. Worked on this a bit more this afternoon. Updating here in the form of an image showing three options and explanations for each. My personal preference is option 3. 
![cfd nav](https://cloud.githubusercontent.com/assets/1450173/26425093/40b0ecc8-4099-11e7-8152-6415dccc72b1.png)

There may need to be more exploration beyond the bullet / cards. I'm open to that if neither of these seems to be working

Further, we could explore adding something to the CFD landing page itself, but I'd like to focus on the nav first in case we decide to include a question or two on this in usability testing tomorrow and Fri.

**Comment by noahmanger** [comment]
Ooh nice. Personally I like the third option the most for how it simplifies things and I think you have a point about the label "Advanced data". Might be worth flipping the order of the two callouts.

**Comment by jenniferthibault** [comment]
I'm also drawn to the third, and would be into seeing what it's like to drop the "Advanced data" title.

Just to surface, why we added the "Advanced data" heading originally with intentionality—more beginner users were navigating to the data tables directly before and feeling like they didn't understand what they were seeing. More simply, these data tables aren't exactly useful for _everyone_ so it was helpful to indicate that it would be more relevant for those who were particularly skilled, while hopefully driving more novice users to the data landing page, where they could poke around the total/summary graphics before diving in.

By adding landing pages for each activity type before the data tables, I think we've done some to alleviate the novice shock, but would like to see the context about the data that we intended to be there actually appear on those pages to help people out more—especially if we drop the "advanced" category from the menu as a first stage indicator. Probably a separate issue, but one that isn't tracked now that would be more important if we are changing the framing before someone clicks into the section before knowing what kind of thing they're looking at.

![screen shot 2017-05-24 at 6 59 58 pm](https://cloud.githubusercontent.com/assets/11636908/26429129/482f3e2c-40b3-11e7-8b46-c1f971185402.png)

**Comment by nickykrause** [comment]
ooohhh, yes, @jenniferthibault, thank you for this explanation. I had forgotten about the issue of users wanting "context" for what they were looking at on data tables, and that this was one reason for moving folks through the advanced data page.

The third menu option that I was proposing still assumed that we would push people to this intermediary `advanced data` page first, so we wouldn't be losing the stop-off altogether, but we'd be losing the general link to the `advanced data` page from the menu. You probably understood that, but I just wanted to make it clearer, in case it was vague.

I can really go either way on keeping the advanced data link (so, option 2 or option 3). It sounds like we're leaning toward option 3, though?

**Comment by nickykrause** [comment]
Quick update on this:

We had another user in testing today who struggled to answer the following question:

> If you wanted to know who the actor George Clooney donated money to in the last presidential election, what would you do to find out?

She went to the CFD menu and didn't see anything she liked. She didn't try to use the site-wide search, and continued browsing around through the menus, and eventually the homepage, where she didn't notice the link to individual contributors. Eventually, I asked her about her first impulse to look in the CFD menu and if she could tell me more about what she saw there. She said:

> Well, he’s not a candidate, and not a committee. I'm looking for where I could put his name where I could start pulling him up.

I asked: "Is there anything specific you're looking for?"
> I can't think of the word I would look for. I was looking for “individual contributions”. Not sure it would be easy for my students to find. 

At this point, I showed her menu option 2, and she thought the card for individual contributors was helpful. She speculated that she would have clicked it and that the wording would make it easy even for her students to understand, if she sent them here with a similar task.

(She teaches a US gov't class)

**Comment by nickykrause** [comment]
Another update from testing:

Asked the user the same question as last time:

> If you wanted to know who the actor George Clooney donated money to in the last presidential election, what would you do to find out?

She looked at each of the menu flyouts, starting with `CFD`, and she said **"I'm just looking for a place that seems like I could input a person's name."** She then added that she was looking for something about "individuals". 

She concluded that nothing in the menus would help her and seemed to want help. I prompted her toward the sitewide search bar and asked if she noticed it or if there was a reason it didn't appeal to her. She said she had noticed it, but that she doesn't typically "trust" sitewide search bars, "because they could be searching anything." She said she prefers search bars that are labeled with exactly what they are doing, like the bars that say `Committee name or ID`. Regarding the site-wide search, she added that "assumes [she] can't put a person's name in that box and expect it to work."

---

**These past two tests add more evidence for highlighting `Individual Contributions` more clearly and easily for users.** However, I think this most recent one _also_ adds to the signal I mentioned above as point number 3:

> 🤔 3. Some users have expressed hesitation about the site-wide search on the grounds that they don't "trust" site-wide searches, but they have not expressed (at least in the sessions I've run) concern with the CFD search or the legal search. Users seem to view these searches differently (possibly as databases) than they view the site-wide search, and they often seem to prefer them.

> 💡 Make the "database searches" easier to reach

That said, I have logged this point about the "trustworthiness" of the search bar as its own testing issue, so that we can separate it from the question of making individual contributions easier to find: https://github.com/18F/fec-testing/issues/69

**Comment by noahmanger** [comment]
So do we have a direction to take with this for implementation?

**Comment by nickykrause** [comment]
I think so, but I'm not sure what @jenniferthibault thinks. I'm ok with trying option 2 or option 3, with a preference for option 3.

**Comment by jenniferthibault** [comment]
Oh I'm sorry! I hadn't realized you were waiting on me. 3 is 👍 but I would like to make an icon for the individual contributions, the one used as placeholder is what we've considered a "candidate" before.

I can make that part of the implementation issue to keep this going.

**Comment by noahmanger** [comment]
Implementation happening in https://github.com/18F/fec-style/issues/715 . Closing.
